src/types/reduxActions.js

```diff
@@ -157,6 +157,7 @@ export type Action =
     }
   | { type: 'IS_CHECKING_HANDLE_AVAILABILITY', data: boolean }
   | { type: 'LOGOUT', data: { username?: string } }
+  | { type: 'SHOW_MAIN_APP' }
```

Comment:
We have another section for actions with no payload, up above. Please move this up there, sorted alphabetically.